lang/en/docs/software-directory/scripting/jupyter-lab/overview.md

```diff
@@ -2,7 +2,7 @@
 
 JupyterLab [^1] is the next-generation user interface for Project Jupyter [^2] offering all the familiar building blocks of the classic Jupyter Notebook [^3] in a flexible and powerful user interface. JupyterLab will eventually replace the classic Jupyter Notebook.
 
-We include support for **Jupyter Lab** within our platform, using the operating system default interpreter version 2.7.5 for Python.
+We include support for **Jupyter Lab** versions 3.0.3 and 0.33.12 within our platform, using Python version 3.8.6.
```

Comment:
We actually have both 2.7.5 and 3.8.6 now. I'd just put a link to the supported versions of python here instead, and track it there. Where is that page btw - somewhere in the software directory?

Reply:
If you remember, we never found a way to select the version of python for the jupyterlab workflows, and rupy is currently wired to activate the python/3.8.6 module when executing.
See: https://github.com/Exabyte-io/exabyte-rupy/blob/dev/src/software/scripting/jupyter_lab/unit.py